astropy/utils/tests/test_subprocess.py

```diff
@@ -0,0 +1,12 @@
+from ...tests.helper import catch_warnings
+from ..exceptions import AstropyDeprecationWarning
+
+
+def test_import_warning():
+    with catch_warnings() as w:
+        from ..compat import subprocess
+
+    assert len(w) == 1
+    assert w[0].category == AstropyDeprecationWarning
+    assert w[0].message.args[0] == ("astropy.utils.compat.subprocess is now deprecated "
+                                    "- use the Python subprocess module directly instead")
```

Comment:
I really don't think this test is necessary.  All it's testing is that warnings work.  I can take it or leave it though.